### PR TITLE
Socint 140

### PR DESCRIPTION
**Pull request by KieranWardle**
Moved PubSub helper to its own package 
https://collaborate2.ons.gov.uk/jira/browse/SOCINT-140

### CODE REVIEW

File: util-framework/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# ONS SDC Integrations Event Publisher
```
Comment (rpcatling) [inline util-framework/README.md:1]: incorrect title

File: util-framework/pom.xml [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  <parent>
+    <groupId>ons.sdc.int.common</groupId>
+    <artifactId>common</artifactId>
+    <!-- version auto updated by Travis mvn release -->
+    <version>1.0.45-SNAPSHOT</version>
+  </parent>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
```
Comment (rpcatling) [inline util-framework/pom.xml:20]: not needed

File: util-framework/pom.xml [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+        </exclusion>
+      </exclusions>
+    </dependency>
+
+    <dependency>
+      <groupId>org.projectlombok</groupId>
+      <artifactId>lombok</artifactId>
+    </dependency>
+
+    <dependency>
```
Comment (rpcatling) [inline util-framework/pom.xml:35]: not needed

File: util-framework/pom.xml [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-collections4</artifactId>
+    </dependency>
+
+    <dependency>
```
Comment (rpcatling) [inline util-framework/pom.xml:40]: not needed

File: util-framework/pom.xml [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+      <groupId>net.logstash.logback</groupId>
+      <artifactId>logstash-logback-encoder</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>spring-cloud-gcp-starter-pubsub</artifactId>
+    </dependency>
+
+    <dependency>
```
Comment (rpcatling) [inline util-framework/pom.xml:55]: not needed

File: util-framework/pom.xml [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+      <version>${project.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>ons.sdc.int.common</groupId>
+      <artifactId>event-publisher</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+    <dependency>
```
Comment (rpcatling) [inline util-framework/pom.xml:74]: not needed

File: util-framework/pom.xml [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+      </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+      </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
```
Comment (rpcatling) [inline util-framework/pom.xml:106]: surefire not needed

**Review by rpcatling — CHANGES_REQUESTED** [review]

**Review by rpcatling — APPROVED** [review]

**Review by PeterBochel — APPROVED** [review]

**Pull request merged**